MaterialDesignThemes.Wpf/PackIcon.cs

```diff
@@ -17,9 +17,11 @@ public class PackIcon : PackIconBase<PackIconKind>
         static PackIcon()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(PackIcon), new FrameworkPropertyMetadata(typeof(PackIcon)));
-        }     
+        }
 
         public PackIcon() : base(PackIconDataFactory.Create)
-        { }    
+        {
+            FlowDirection = FlowDirection.LeftToRight;
```

Comment:
Icons are mirrored in RTL mode. which is not exactly the way they were meant to be shown. So I had to give them LTR FlowDirection directly